### ISSUE TITLE
Standard libraries for working with common languages 

### ISSUE THREAD

**Issue by Cypher1** [project-oak/arcsjs-provable#48]
## Current state
All Ibis recipes must include all types that they may use.

## Future work
Ibis should have a set of 'libraries', files that describe the type systems of languages commonly used with `ArcsJs` (e.g. Javascript, Rust, Type Script).

These should be able to be selected and included when working with Ibis recipes to reduce the work needed to use Ibis to just describing particles, stores and their connections (and not the host language).

Depends on #47
